src/app/components/users/UsersNotInTeam.test.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import UsersNotInTeam from "./UsersNotInTeam";
+import { mount } from "enzyme";
+import renderer from "react-test-renderer";
+import { WrapperComponent } from "../../../tests/test-utils";
+
+let dispatchedActions = [];
+const mockDispatch = event => {
+    dispatchedActions.push(event);
+};
+
+describe("UsersNotInTeam", () => {
+    const UsersNotInTeamProps = {
+        dispatch: mockDispatch,
+        loading: false,
+    };
+    describe("Given valid props on initial load", () => {
+        const component = mount(
+            <WrapperComponent>
+                <UsersNotInTeam {...UsersNotInTeamProps} />
+            </WrapperComponent>
+        );
+        it("renders the relevant components to screen", () => {
```

Comment:
can you please assert the props you passed to the component in line 20 . Also I can't see the test for other props passed to the component: 
`addUserToTeam: PropTypes.func,
    usersNotInTeam: PropTypes.arrayOf(PropTypes.object),
    allPreviewUsers: PropTypes.arrayOf(PropTypes.object),` 
    there is so much logic in this simple component it needs to be tested